support/wolfssl.pc

```diff
@@ -1,4 +1,4 @@
-prefix=/usr/local
+prefix=/usr
```

Comment:
yes, indeed. This file is automatically generated by `configure`. I committed after configuring with --prefix=/usr. Perhaps this file should not be included in the repo?